src/rascal/math/utils.hh

```diff
@@ -189,6 +189,39 @@ namespace rascal {
 
       return rel_diff;
     }
+
+    /**
+     * Routine to compute the relative difference between 2 double.
+     * The return value depend on the closeness to zero:
+     * + if ref and test are not zero, then return relative error
+     * + if ref and test are within epsilon, then return 0.
+     * + if ref is within epsilon and test not, then return absolute error
+     *
+     * @param reference the double with the reference values
+     * @param test the double with the  values to check
+     * @param epsilon the threshold defining which values are effectivelly zero
+     * @param delta maximum relative error threshold
```

Comment:
> Maximum error threshold for what?

that's the tolerance on the relative error between test and reference. We agreed that it was counter productive to throw an error here so we just print a warning since we don't have a better alternative at the moment.

Reply:
Ok, so it's the `maximum relative error threshold to print warning`? Could the documentation be made to reflect that?